cgc/teltale.cwl

```diff
@@ -0,0 +1,24 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  sbg: 'https://sevenbridges.com'
+baseCommand: []
+inputs:
+  - id: input_file
+    type: File
+    inputBinding:
+      position: 1
+    label: Input BAM file
+    'sbg:fileTypes': BAM
```

Comment:
Is there a CWL generic option to do this also? Something that would enforce a ".bam" extension.

Reply:
Not that I have seen, but I really haven't been looking for anything that does enforcement....that `sbg:fileTypes` just allows the UI to do some things as far as I can tell.  But I can put .bam at the end of any file and CGC will treat it as such

Reply:
I was thinking of non-CGC use cases, but let's leave it as is.

Reply:
Either way, Ill investigate.